server/test/client.jl

```diff
@@ -0,0 +1,233 @@
+# start Wflow ZMQ server (@async)
+@async begin
+    WflowServer.start()
+end
+
+# Connecting to the Wflow ZMQ Server
+context = Context()
+socket = Socket(context, REQ)
+ZMQ.connect(socket, "tcp://localhost:5555")
+
+function request(message)
+    ZMQ.send(socket, JSON3.write(message))
+    ret_value = JSON3.read(ZMQ.recv(socket), Dict)
+    return ret_value
+end
+
+@testset "Client requests" begin
```

Comment:
Including `runtests.jl` all tests are wrapped in two 1-size testset right? We could drop this one.

Reply:
Yes, I did change this (with combination of two different tests because of depending variables).